tests/integration/conftest.py

```diff
@@ -0,0 +1,52 @@
+import os
+import uuid
+
+import pytest
+from aiodynamo.client import Client
+from aiodynamo.credentials import Credentials
+from aiodynamo.models import KeySchema, KeySpec, KeyType, Throughput, WaitConfig
+from yarl import URL
+
+
+@pytest.fixture
+def table_name_prefix() -> str:
+    return os.environ.get("DYNAMODB_TABLE_PREFIX", "")
+
+
+@pytest.fixture
+def endpoint():
+    if os.environ.get("TEST_ON_AWS", "false") == "true":
```

Comment:
how about 
```python3
if os.environ.get("TEST_ON_AWS"):
```

Reply:
I think it's rare enough that being super explicit about what you're doing is fine. this is the tests after all.